Snippets/ASB/ASB_7/Performance.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using NServiceBus;
+
+class Performance
+{
+    async Task SendPerformance()
+    {
+        var endpointConfiguration = new EndpointConfiguration("whatever");
+        var endpointInstance = await Endpoint.Start(endpointConfiguration).ConfigureAwait(false);
+
+        #region asb-slow-send
+
+        for (var i = 0; i < 100; i++)
+        {
+            // by awaiting each individual send, no client side batching can take place
```

Comment:
@ramonsmits In the samples I have written all sends this way, even the ones inside handlers, because it is also applicable when  sendoptions ask for immediate dispatch. Only for default dispatch the await doesn't matter.